client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/Forms/ExposureForm/ExposureFormStyles.ts

```diff
@@ -43,6 +43,9 @@ const useStyles = makeStyles({
     },
     errorLabel: {
         color: theme.palette.error.main
+    },
+    placeTypeSpace: {
+        margin: '30px 0 20px'
```

Comment:
add 0 at the end instead of nothing

Reply:
there is no need to add... if you don't put its add the same value as the second value